README.md

```diff
@@ -1,123 +1,15 @@
-# Differ.rs
-[![License](https://img.shields.io/badge/license-MIT%20%26%20Apache%202.0-green)](#license)
-[![CI](https://github.com/nlp-rs/differ.rs/actions/workflows/main.yml/badge.svg)](https://github.com/nlp-rs/differ.rs/actions/workflows/main.yml)
-[![Security audit](https://github.com/nlp-rs/differ.rs/actions/workflows/security-audit.yml/badge.svg)](https://github.com/nlp-rs/differ.rs/actions/workflows/security-audit.yml)
-> warning: **Differ.rs is currently experimental**
-This crate provides edit distance, delta vectors between 2 words, and lets you apply delta vectors in order to transform words.
+# differ.rs
 
-## Install
-```shell
-cargo add differ-rs
-```
-or, simply add the following string to your Cargo.toml:
+A natural language processing library, written in Rust.
```

Comment:
The repository itself is not a library, but a repository of related libraries for diffing. Could you make it a bit more explicit here?